data/mods/Magiclysm/items/enchanted_melee.json

```diff
@@ -89,6 +89,24 @@
     "proportional": { "price": 6.0, "price_postapoc": 6.0, "bashing": 1.2, "cutting": 1.2, "weight": 0.8 },
     "relative": { "to_hit": 2 }
   },
+  {
+    "type": "GENERIC",
+    "id": "sledge_heavy_plus_one",
+    "copy-from": "hammer_sledge_heavy",
+    "looks_like": "hammer_sledge_heavy",
+    "name": "heavy sledge hammer +1",
+    "proportional": { "price": 3.0, "price_postapoc": 3.0, "bashing": 1.1, "cutting": 1.1, "weight": 0.9 },
+    "relative": { "to_hit": 1 }
+  },
+  {
+    "type": "GENERIC",
+    "id": "sledge_heavy_plus_two",
+    "copy-from": "hammer_sledge_heavy",
+    "looks_like": "hammer_sledge_heavy",
+    "name": "heavy sledge hammer +2",
+    "proportional": { "price": 6.0, "price_postapoc": 6.0, "bashing": 1.2, "cutting": 1.2, "weight": 0.8 },
```

Comment:
These could use a bigger increment in `price` and `price_postapoc` IIRC some of the +1 or +2 tools are extremely cheap, I think I'll check the rest of them later.
Also 
![image](https://user-images.githubusercontent.com/58050969/112738251-29a75880-8f2f-11eb-9c1c-6a68f0619b3e.png)

Reply:
I oriented myself on the pre-existing magical weapons (that all use the same proportional line) but that's probably a placeholder until someone puts more effort into it
the last one is neat; I need to work on some prs now :D